tensorflow/tools/ci_build/ci_parameterized_build.sh

```diff
@@ -50,9 +50,18 @@
 #                      builds where the tests cannot be run in parallel due to
 #                      resource contention (e.g., for GPU builds)
 #   TF_BUILD_TEST_TUTORIALS:
-#                      Perform tutorials test (Applicable only if
-#                      TF_BUILD_IS_PIP is PIP or BOTH).
-#                      See build/test_tutorials.sh
+#                      If set to any non-empty and non-0 value, will perform
+#                      tutorials tests (Applicable only if TF_BUILD_IS_PIP is
+#                      PIP or BOTH).
+#                      See builds/test_tutorials.sh
+#   TF_BUILD_DOCKER_TEST:
```

Comment:
Do you really want to put the release docker testing into this script? There is already way too many things here, is it not?